Gruntfile.js

```diff
@@ -293,14 +324,25 @@ module.exports = function(grunt) {
       }
     },
     usebanner: {
```

Comment:
We can run the unminified code through uglify beautifier if we wanted to. But makes sense to keep banner.
Does uglify create sourcemaps? If so, this could mess up with that if it is run after uglify.